package.json

```diff
@@ -65,13 +65,13 @@
         "prettier": "^3.1.0",
         "react": "^18.2.0",
         "react-dom": "^18.2.0",
-        "react-router-dom": "^5.3.4",
+        "react-router-dom": "^6.0.0",
         "styled-components": "^5.3.3"
```

Comment:
Styled components also needs to be updated, see our boilerplate https://github.com/pluginpal/strapi-plugin-boilerplate/blob/main/package.json#L67

Also all these dependencies (including @strapi/strapi) shouldn’t be direct dependencies, but rather peerDependencies. Could you update that as well? Please again take the boilerplate as a example.